src/mono/wasm/debugger/tests/debugger-datetime-test.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
What is the purpose of all these tests? I see we're not doing any Asserts and just Console.WriteLine.

Reply:
> @thaystg can you follow up on @safern s comments in your unhandled exception PR?

Yes.

Reply:
@tqui8 can you follow up and make these tests actually assert now the that we should match the platform.

Reply:
@safern These are the "test apps", which the debugger unit tests debug, by setting breakpoints in the test app code, and inspecting the state.

Reply:
For example - https://github.com/thaystg/runtime/blob/abdc724d7cd76fea608198826e3391bbe830034d/src/mono/wasm/debugger/DebuggerTestSuite/ArrayTests.cs

Reply:
thanks @radical -- @thaystg reached out offline and explained... thanks for explaining 😄